samples/convert_miqa_import_csv_format.py

```diff
@@ -0,0 +1,65 @@
+import pandas
+from pathlib import Path
+import sys
+
+for file_name in sys.argv[1:]:
```

Comment:
I've been quite impressed with [djclick](https://github.com/GaretJax/django-click) for writing scripts like this. It basically lets you register your own manage.py commands, which isn't immediately useful for this since you don't need the django settings or models or anything, but it might be a good central place to keep scripts.

See https://github.com/dandi/dandi-api/tree/master/dandiapi/api/management/commands for some examples